.travis.yml

```diff
@@ -63,7 +61,7 @@ matrix:
         - export PATH="/c/Python37:/c/Python37/Scripts:$PATH"
         - python -m ensurepip
         - pip install virtualenv
-        - virtualenv $HOME/venv
+        - python -m venv $HOME/venv
```

Comment:
I'm surprised that the command here isn't `python3`, I suppose it must be 3 otherwise it would fail to install dependencies.